react/start/runtime.ts

```diff
@@ -0,0 +1,45 @@
+const getValue = (element: HTMLTemplateElement) => {
+  if (typeof element.content === 'undefined') {
+    return element.innerHTML
+  }
+
+  let value = ''
+  const childNodes = element.content.childNodes
```

Comment:
```suggestion
  // Using nodeValue instead of innerHTML because it's faster
  // https://stackoverflow.com/questions/21311299/nodevalue-vs-innerhtml-and-textcontent-how-to-choose/21311670#21311670
  // The reason for the loop is because large texts are broken into multiple childNodes
  const childNodes = element.content.childNodes
```